app/views/katello/api/v2/packages/compare.json.rabl

```diff
@@ -0,0 +1,10 @@
+object false
+
+extends "katello/api/v2/common/metadata"
+
+child @collection[:results] => :results do
+  extends 'katello/api/v2/packages/show'
```

Comment:
we should use 'katello/api/v2/packages/base' here, because the show template makes a call out to pulp to get more details, its likely the same with errata and possibly other types as well.  I'd use base across all content types

Reply:
it looks like the one that is there today (errata), uses show, but it doesn't actually have a base.   All the other types do have a base.json.rabl as far as i can tell, and i' recommend that over show.json.rabl

Reply:
docker_manifests and ostree_branches also do not have a base.  However, I will change the others to use base instead of show.